docs/features/software-catalog/catalog-customization.md

```diff
@@ -140,53 +300,40 @@ export const EntitySecurityTierPicker = () => {
 };
 ```
 
-Now we can add the component to `CustomCatalogPage`:
-
-```tsx
-export const CustomCatalogPage = ({
-  columns,
-  actions,
-  initiallySelectedFilter = 'owned',
-}: CatalogPageProps) => {
-  return (
-    {/* ... */}
-        <EntityListProvider>
-          <CatalogFilterLayout>
-            <CatalogFilterLayout.Filters>
-              <EntityKindPicker initialFilter="component" hidden />
-              <EntityTypePicker />
-              <UserListPicker initialFilter={initiallySelectedFilter} />
-              {/* highlight-add-next-line */}
-              <EntitySecurityTierPicker />
-              <EntityTagPicker />
-            <CatalogFilterLayout.Filters>
-            <CatalogFilterLayout.Content>
-              <CatalogTable columns={columns} actions={actions} />
-            </CatalogFilterLayout.Content>
-          </CatalogFilterLayout>
-        </EntityListProvider>
-    {/* ... */}
-};
-```
-
-Finally, we can apply our new `CustomCatalogPage`.
+Now we can add the component to `CatalogIndexPage`:
 
 ```tsx title="packages/app/src/App.tsx"
+{
+  /* highlight-add-start */
+}
+import { DefaultFilters } from '@backstage/plugin-catalog-react';
+{
+  /* highlight-add-end */
+}
```

Comment:
```suggestion
{/* highlight-add-start */}
import { DefaultFilters } from '@backstage/plugin-catalog-react';
{/* highlight-add-end */}
```

Reply:
Fixed, seems every time my editor applies the formatting it does this, not sure why just here and not the other cases 🤔